runtime_builders/publish_builders.py

```diff
@@ -31,18 +32,20 @@ def main():
                         required=True)
     args = parser.parse_args()
 
-    failures = 0
     try:
         for filename in os.listdir(args.directory):
             filepath = os.path.join(args.directory, filename)
             if filepath.endswith('.json'):
                 with open(filepath, 'r') as f:
                     config = json.load(f)
-                    for builder in config['releases']:
-                        staged_builder = builder['path']
-                        for tag in builder['tags']:
-                            failures += _copy(staged_builder, tag)
-        return failures
+                    project_name = config['project']
+                    latest = config['latest']
+                    prefix = builder_util.RUNTIME_BUCKET_PREFIX
+                    latest_file = latest.lstrip(prefix) \
```

Comment:
lstrip doesn't quite do what you want I think. You might be better off with a regex here.